blueprints/validator-test/index.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
+var existsSync = require('exists-sync');
+var walkSync = require('walk-sync');
+
+module.exports = {
+  description: 'Generates a validator unit test',
+
+  _getFilesPath: function() {
+    var type;
+    if ('ember-cli-mocha' in this.project.addonPackages) {
+      type = 'mocha';
+    } else if ('ember-cli-qunit' in this.project.addonPackages) {
+      type = 'qunit';
+    }
```

Comment:
If the app isnt using mocha or qunit, we should either default to qunit or exit out with a message.